Docker Install Update (#4248)
mattermost/docs: source/install/prod-docker.rst

      sudo curl -L "https://github.com/docker/compose/releases/download/$dockerComposeVersion/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
      sudo chmod +x /usr/local/bin/docker-compose

3. **Deploy the Mattermost Docker** 

> Find out the :code:`uid` (user ID) and :code:`gid` (group ID) of the user to run the docker, by running:

Reviewer: ```suggestion
You can get the :code:`uid` (user ID) and :code:`gid` (group ID) of the Docker user by running:
```

   .. code:: bash
   
       id <username>

Replace :code:`<username>` with the actual username. The following setup assumes the result above is :code:`uid=1000` and :code:`gid=1000`.

> Setup by running:

Reviewer: ```suggestion
Set up using:
```
